bin/prepare-kubectl

```diff
@@ -1,6 +1,11 @@
 #!/bin/bash
 
+  . k8s-read-config "$@"
```

Comment:
The example should be updated for this change as well as they pass in config files and do not have a default file to fall back to. This I think is a breaking change so should have a bumped major number on release.

Reply:
I'm unsure why this line was added.  It was in the branch when I started working on it.
Does it appear to be necessary to you? I'd prefer to not introduce a breaking change if we don't need to.

Reply:
I'd mark is as a "very good to have". This script now uses more vars and without reading the config it means they can't be specified in the *.config. Breaking change should be fine as long as we document it in the release notes.